colossus-tests/src/it/scala/colossus/RedisSetITSpec.scala

```diff
@@ -22,7 +22,7 @@ class RedisSetITSpec extends BaseRedisITSpec {
         (x, y)
       }
 
-      res.futureValue must be ((3, Set(val1, val2, val3)))
+      res.futureValue must be((3, Set(val1, val2, val3)))
```

Comment:
I prefer the space since we're doing that with must, thoughts?

Reply:
This is scalafmt. Since there are brackets involved, I'm guessing it removes the space. Personally, I prefer to use assert(...) instead of must/must be/etc.